tests/routes/web/home.spec.ts

```diff
@@ -21,10 +26,19 @@ describe('Home Routes', () => {
   it('should GET / route works', async () => {
     const options = {
       method: 'GET',
-      url: '/',
+      url: webRoutePaths.home,
     };
 
     const response = await server.inject(options);
+    const rawHTML = response.payload;
+    const parser = new DOMParser();
+    const document = parser.parseFromString(rawHTML, 'text/html');
     expect(response.statusCode).toEqual(200);
+    expect(
```

Comment:
Can you group the assertions in a test case and group the test cases in a test suite.

Reply:
Yes, now we have grouped the test cases.